nixos/tests/ly.nix

```diff
@@ -0,0 +1,42 @@
+{ system ? builtins.currentSystem,
+  config ? {},
+  pkgs ? import ../.. { inherit system config; }
+}:
+
+with import ../lib/testing-python.nix { inherit system pkgs; };
+
+let
+  inherit (pkgs) lib;
+
+  tests = {
+    default = {
```

Comment:
No need to do this `tests.default` thing. Just follow how other tests are being defined.

Reply:
BTW the test is stuck here. Over 15 minutes without any news.

Reply:
This pr is working in progress. The module cannot work correctly now.